src/Compilers/VisualBasic/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.vb

```diff
@@ -15,6 +15,8 @@ Imports Microsoft.CodeAnalysis.VisualBasic.Symbols.Metadata.PE
 Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
 Imports Roslyn.Test.Utilities
 
+#Disable Warning SYSLIB0050 ' 'TypeAttributes.Serializable' is obsolete
```

Comment:
What's the context for this? I'm not seeing that attribute as obsolete on [https://learn.microsoft.com/en-us/dotnet/api/system.reflection.typeattributes?view=net-8.0](https://learn.microsoft.com/en-us/dotnet/api/system.reflection.typeattributes?view=net-8.0)
If it is obsolete, are we supposed to use something else instead?

Reply:
It's suppressed at other places that were upgraded to net8 previously, e.g.:

https://github.com/dotnet/roslyn/blob/32b3fc05bff6daca73d91c3ab162c2aa7a3cb7aa/src/Compilers/VisualBasic/Portable/Symbols/Metadata/PE/PENamedTypeSymbol.vb#L223

See also comment in https://github.com/dotnet/roslyn/pull/68627/files